cmd/dockerd/daemon.go

```diff
@@ -484,7 +484,7 @@ func initRouter(s *apiserver.Server, d *daemon.Daemon, c *cluster.Cluster) {
 		image.NewRouter(d, decoder),
 		systemrouter.NewRouter(d, c),
 		volume.NewRouter(d),
-		build.NewRouter(dockerfile.NewBuildManager(d)),
+		build.NewRouter(manager.NewBuildManager(d)),
```

Comment:
I think `build.NewRouter` should just take `build.TagSquashBackend=d` as a second parameter. Same way `systemrouter` and `networkrouter` do it.